### PR TITLE
Encode `uint64` as LE

### PR DESCRIPTION
**Pull request by kasperisager**
See https://github.com/hypercore-protocol/hypercore-next/pull/62.

### CODE REVIEW

**Comment by mafintosh** [comment]
Awesome! Do you mind adding a short line in the readme about how Hypercore 9 uses v2 of this and 10 uses v3?

**Comment by kasperisager** [comment]
On a related note, should I remove the functions not used by Hypercore 10?

**Comment by mafintosh** [comment]
Good idea, then we can add them back as we need them without "breaking" anything.

File: index.js [hunk 1]
```diff
@@ -149,33 +87,8 @@ exports.tree = function (roots, out) {
...
-  c.uint64.encode({ start: 32, end: 40, buffer: out }, length)
-
-  return out
-}
-
-exports.randomBytes = function (n) {
-  const buf = b4a.allocUnsafe(n)
-  sodium.randombytes_buf(buf)
-  return buf
-}
```
Comment (mafintosh) [inline index.js:167]: this one we should keep, used somewhere in some dep

File: index.js [hunk 2]
```diff
@@ -149,33 +87,8 @@ exports.tree = function (roots, out) {
...
   return digest
 }
-
-if (sodium.sodium_free) {
-  exports.free = function (secureBuf) {
-    if (secureBuf.secure) sodium.sodium_free(secureBuf)
-  }
-} else {
-  exports.free = function () {}
-}
```
Comment (mafintosh) [inline index.js:181]: let's keep this one also

**Pull request merged**